docs/src/architecture/accounts.md

```diff
@@ -52,6 +52,27 @@ Interface for accounts. In Miden every account is a smart contract. It has an in
 
 *Note: Since code in Miden is expresed as MAST, every function is a commitment to the underlying code. The code cannot change unnoticed to the user because its hash would change. Behind any MAST root there can only be `256` functions*
 
+#### Example Account Code
+Currently, Miden provides two standard implementations for account code. 
+
+A basic user account exposes the following code as its interface.
+
+```
+    use.miden::contracts::wallets::basic->basic_wallet
+    use.miden::contracts::auth::basic
+
+    export.basic_wallet::receive_asset
+    export.basic_wallet::send_asset
+    export.basic::auth_tx_rpo_falcon512
+
+```
+
+That means, [note scripts](https://0xpolygonmiden.github.io/miden-base/architecture/notes.html#note-scripts) or transaction scripts can call those functions and in doing so, the account can receive and send assets and authenticate the transaction. It is important to know, that without correct authentication, a note cannot successfully invoke receive and send asset.
+
+There is also a standard for a [basic fungible faucet](https://github.com/0xPolygonMiden/miden-base/blob/main/miden-lib/asm/miden/contracts/faucets/basic_fungible.masm). It exposes two functions `distribute` and `burn`. The first function can only be called by the faucet owner, otherwise it fails. `burn` can be called by anyone to burn the tokens that are contained in a note.
```

Comment:
I would add the code as example for the fungible faucet as you did with the basic user account.

Reply:
I would explain that the faucet owner calls distribute passing a user ID and minting a new note destined for a user to consume. ( Not said like this but this is the global idea )

Reply:
I would change the first line to: `A standard for a basic fungible faucet also exists`